### ISSUE TITLE
Issue running Pequeno Quick start on Windows

### ISSUE THREAD

**Issue by ashleydavis** [signalkuppe/pequeno#2]
Hi, this project looks great!

I'm having a small problem getting started.

I've followed steps 1 and 2 of the "Quick start" and I get the following warning:

```bash
info: Nothing to copy to .site
warn: No pages found, start adding .jsx templates in src\pages
```

I've created an example here to show the problem:
https://github.com/ashleydavis/pequeno-test

Please let me know how to get past this issue! I'm keen to try out my static site through Pequeno.

**Comment by signalkuppe** [comment]
hi @ashleydavis. Just cloned your repo and everything works fine (npm install and npm start)

```js
> pequeno-test@1.0.0 start /Users/signalkuppe/Desktop/pequeno-test
> pequeno --serve

info: Nothing to copy to .site
info: writed .site/empty.html
info: 🚀 Build finished: writed 1 files in 0.3 seconds (3 pages per second)
[Browsersync] Access URLs:
 --------------------------------------
       Local: http://localhost:8080
    External: http://192.168.1.186:8080
 --------------------------------------
          UI: http://localhost:3001
 UI External: http://localhost:3001
 --------------------------------------
```

Can you give me more infos about your env?

**Comment by ashleydavis** [comment]
Thanks for the quick reply.

It's very strange.

I just cloned fresh and it still doesn't work for me.

I've tried it with Node.js v14.16.0 and v16.13.1.

I'm on Windows 10.

I can also try it later on Linux and MacOS. What platform do you test on?

According to `npm list` I'm using Pequeno 0.0.31.

Let me know if there's more information I can give you.

**Comment by ashleydavis** [comment]
My example works on MacOS!

It must be just Windows where it doesn't work. You might be assuming something about the file system that doesn't hold true for Windows.

Do you plan to support Windows?

**Comment by signalkuppe** [comment]
Ok, thanks for reporting this, will check

**Comment by signalkuppe** [comment]
@ashleydavis pushed a fix in ```windows-path-fixes``` branch.
Can you check if this solves your problem?

ps: tested on windows 11 and works fine.

**Comment by ashleydavis** [comment]
Thanks for looking into it! Let me know when you publish a new version and I'll install and confirm if it fixes my issue.

**Comment by signalkuppe** [comment]
@ashleydavis released 0.0.32

**Comment by ashleydavis** [comment]
Thanks so much! I can confirm that it works now.

You might be interested to know why I was looking at Pequeno in the first place...

I'm building my own little static web site for a video portal (powered by YouTube) as a way to showcase my own videos. I had started with Eleventy, but when I realised I wanted to use React and that Eleventy didn't allow for it I started investigating other options (at this stage I'd only just started development and it's not a huge site anyway).

I tried Parcel + React, but Parcel doesn't have the static generation features I needed. I tried React-Static, which was ok but seemed to complicated and didn't give me control of the html file (I'll log a separate issue for this and see if you want to deal with it).

Then I tried Pequeno and couldn't get it to work (thanks for that!). After that I played around with making my own static site generator build pipeline.  That was a bit of fun, but then I realised I had take it way too far!

Ultimately I came back to Eleventy and found the best way forward was to make a React plugin for Eleventy. That allowed me to keep the work I'd already done and move forward having React support in Eleventy.

My work is here if you are interested:
https://github.com/ashleydavis/eleventy-plugin-react-static

Thanks so much for Pequeno, it looks great and I'll consider it next time I'm making a static site.
